diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index b547b8991f665b17f3ac691d86dee5200d94035c..01be49b3a49f5a6a476eac5f2fa63f74e55b1800 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2796,6 +2796,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
