diff --git a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
index a0d015934e052aa9b5c36815e4c53b38bcb83b61..74b98f7c86e669bc81f2927fd4fee78211bdf2fb 100644
--- a/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
+++ b/chrome/browser/ui/views/profiles/avatar_toolbar_button.h
@@ -119,6 +119,7 @@ class AvatarToolbarButton : public ToolbarButton {
   void NotifyShowSigninPausedDelayEnded() const;
 
  private:
+  BRAVE_AVATAR_BUTTON_H
   FRIEND_TEST_ALL_PREFIXES(AvatarToolbarButtonTest,
                            HighlightMeetsMinimumContrast);
 
