diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index 56b29f25ae22885e661447ae9de6ecdeb46c45ab..5647a885a747b550cbb302df6372e7140e4fa537 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -1487,6 +1487,7 @@ source_set("extensions") {
       "//components/enterprise/data_controls",
     ]
   }
+  deps += brave_extensions_resources
 }
 
 proto_library("cws_item_service_proto") {
