diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 9c7f49d48571375689462ae3dfd9109cf6c09bdf..20f2c9a31dc8a6d917198484c78e450aabb9b3e5 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2790,6 +2790,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
