diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index b8937e3140cf6ddf3d1fbe4fb28e169030337f73..0b64f06d860f868dfe5161bfbad4d077a2cb4d7b 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -306,6 +307,7 @@ function createRoutes(): SettingsRoutes {
     }
     // </if>
   }
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
