diff --git a/components/search_engines/BUILD.gn b/components/search_engines/BUILD.gn
index 5ec07405acebd1f2b03ccd71319b49bf2be8fff1..fc7f2256569d049dd630cbb2458d186387b006c3 100644
--- a/components/search_engines/BUILD.gn
+++ b/components/search_engines/BUILD.gn
@@ -253,6 +253,7 @@ json_to_struct("prepopulated_engines") {
   schema_file = "prepopulated_engines_schema.json"
   namespace = "TemplateURLPrepopulateData"
   deps = [ ":search_engine_type" ]
+  additional_sources = brave_components_search_engines_sources
 }
 
 if (is_android) {
