diff --git a/content/common/BUILD.gn b/content/common/BUILD.gn
index fc29dd8b3fdba8260fb373ca2d54552980e5bf98..582a9cc61967482955b593699700719fafedc15c 100644
--- a/content/common/BUILD.gn
+++ b/content/common/BUILD.gn
@@ -69,6 +69,7 @@ source_set("common") {
   visibility = [
     "//content/*",
     "//tools/ipc_fuzzer/message_replay:ipc_fuzzer_replay",
+    "//brave/content/*",
   ]
 
   sources = [
