diff --git a/ios/chrome/app/BUILD.gn b/ios/chrome/app/BUILD.gn
index 27d7306996bf5c9585ec8b5d3f7e517712fdb9f9..eded129c7e7d0cd4fc5be15baeef06c0fb7c9808 100644
--- a/ios/chrome/app/BUILD.gn
+++ b/ios/chrome/app/BUILD.gn
@@ -860,8 +860,7 @@ chrome_app("chrome") {
   # (see https://crbug.com/1394553 for details of the build failure). This
   # is a stop-gap solution.
   if (target_environment == "catalyst" ||
-      ios_provider_target !=
-      "//ios/chrome/browser/providers:chromium_providers") {
+      !ios_chrome_links_with_lottie_framework) {
     assert_no_deps += [
       "//ios/third_party/lottie:lottie+bundle",
       "//ios/third_party/lottie:lottie+link",
