diff --git a/third_party/blink/renderer/core/script/script_loader.cc b/third_party/blink/renderer/core/script/script_loader.cc
index f48f770cfa60ef7c2bab753553cb27c31cac8e18..417261fc0734a85707dc27c4e32c1e8dc585f7d3 100644
--- a/third_party/blink/renderer/core/script/script_loader.cc
+++ b/third_party/blink/renderer/core/script/script_loader.cc
@@ -733,6 +733,7 @@ PendingScript* ScriptLoader::PrepareScript(
                              parser_state, credentials_mode, referrer_policy,
                              fetch_priority_hint, render_blocking_behavior,
                              RejectCoepUnsafeNone(false));
+  BRAVE_SCRIPT_LOADER_PREPARE_SCRIPT
 
   // <spec step="28">Let settings object be el's node document's relevant
   // settings object.</spec>
