diff --git a/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.ts b/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.ts
index 3e13b3d7b77c3ecd00e49aa69f05e971476b3d28..ad19968da1adff3ecd744b45b074eb24812cb6e2 100644
--- a/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.ts
+++ b/ui/webui/resources/cr_elements/cr_toolbar/cr_toolbar.ts
@@ -156,4 +156,4 @@ declare global {
   }
 }
 
-customElements.define(CrToolbarElement.is, CrToolbarElement);
+// customElements.define(CrToolbarElement.is, CrToolbarElement); // We specify our own toolbar
