diff --git a/BUILD.gn b/BUILD.gn
index 0e53f9bc42736f032f2a0c186da8f96c44a59425..d59336ab0cdf346b6434e35cae3c18a6dc11fc09 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -748,6 +748,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3119,6 +3120,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
