diff --git a/BUILD.gn b/BUILD.gn
index 7bdf43e50a85c78a2c870ab1b574e57750620a28..2838ff3aecd2cc1066fa4c6b87a7c5b0616e0f8e 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -748,6 +748,7 @@ config("internal_config_base") {
     "include",
     "$target_gen_dir",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3119,6 +3120,7 @@ v8_header_set("v8_headers") {
     ":cppgc_headers",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
