diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 55116470f0eb403d7ef0a9051f7544d92f5ae89a..c4c63672e717086171ee2733d79f075d9680c2f8 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3016,6 +3016,7 @@ MaybeHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   DCHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
